src/components/KTextArea/KTextArea.vue

```diff
@@ -18,16 +18,28 @@
       </template>
     </KLabel>
 
-    <textarea
-      :id="textAreaId"
-      v-bind="modifiedAttrs"
-      :aria-invalid="ariaInvalid"
-      class="input-textarea"
-      :class="[resizable || isResizable ? 'resizable' : undefined]"
-      :rows="rows"
-      :value="getValue()"
-      @input="inputHandler"
-    />
+    <div
+      class="input-wrapper"
+      :class="{
+        autosize,
+        legacy: !SUPPORT_FIELD_SIZING_CONTENT
+      }"
+      :data-value="SUPPORT_FIELD_SIZING_CONTENT ? null : currValue"
```

Comment:
question: what's the purpose of `data-value` here?

Reply:
It will be referenced by the `attr(data-value)` below.

Reply:
Just a bit concerned about the disconnect between the otherwise unused attribute having an actual purpose here in the style block. 

I'm thinking at a minimum we add context for the relationship somewhere (eg comments, etc.)

Reply:
Added.